### ISSUE TITLE
Daily Reports Chart only shows data from production

### ISSUE THREAD

**Issue by ardichoke** [voxpupuli/puppetboard#381]
Since all other data on the page changes to show data from the currently selected environment, it would make sense for the Daily Reports Chart to also show data from the selected environment. Currently it only shows data from production, or so it seems. We currently have no nodes running against the production environment on our dev puppet 4 server, so there is no data in the chart even when our dev environment is selected and node data shows up everywhere else.

**Comment by ardichoke** [comment]
Correction: the Daily Reports Chart shows the data from whatever environment is configured as the default for puppetboard. I still think when switching environments in the interface, it should update the chart with the data from the selected environment.

**Comment by kirkins** [comment]
What tab are you referring to for daily reports, Overview?

**Comment by ardichoke** [comment]
I have attached a screenshot with the area I'm referring to circled. This screenshot shows our "dev" environment, which is the default environment in the puppetboard configuration, and shows the correct numbers for changed/unchanged/failed. In the second screenshot I have attached, I selected the "production" environment which only has 1 host currently. The bar charts still show the numbers for the "dev" environment when instead it should be showing the numbers for "production"
![puppetboard](https://user-images.githubusercontent.com/112836/27348081-e140dbda-55c0-11e7-9ba8-8a2f948edde2.png)
![puppetboard 1](https://user-images.githubusercontent.com/112836/27348085-e56c58b0-55c0-11e7-9977-5c5a21b4d237.png)

**Comment by mterzo** [comment]
I doubled checked, a change was submitted to fix this: 3b817e0a5ed3e1029a61b55e5c9197d6cb25f30c

This change has not made a pypi release yet.  What version are you on?

I just tested with Safari and the latest head release, and it's working as expected, here's logs from my HTTP server, showing that the ENV is being added to the URL requested to daily charts.

**Default ENV:**
```
10.0.0.6 - - [21/Jun/2017:02:27:02 +0000] "GET /daily_reports_chart.json HTTP/1.1" 200 870 "http://puppetboard.internal.terzo.org/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/603.2.4 (KHTML, like Gecko) Version/10.1.1 Safari/603.2.4"
```
**Using "\*/ALL" ENV**
```
10.0.0.6 - - [21/Jun/2017:02:27:11 +0000] "GET /%2A/daily_reports_chart.json HTTP/1.1" 200 870 "http://puppetboard.internal.terzo.org/%2A/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_5) AppleWebKit/603.2.4 (KHTML, like Gecko) Version/10.1.1 Safari/603.2.4"
```

If you are on the latest version out of github, can you post http logs showing the `GET` for dailychart.  Also what browser you are using, this may not be cross functional?

**Comment by ardichoke** [comment]
We're running the last release version (0.2.2). It looks like that commit was committed after I filed this report (but was never attached to this report).

**Comment by kirkins** [comment]
I'm looking through the issue and thought I'd ping this one.

Seems @ardichoke is saying that a commit made after he filed this fixed the issue?

I'm testing on my setup and I don't see the issue so I'm assuming it's fixed. Can this be closed?

**Comment by mterzo** [comment]
Yup, it's fixed just not in a release yet.  The fixed caused #394

**Comment by mterzo** [comment]
Fixed in 0.3.0
